helm/cilium/values.yaml

```diff
@@ -849,7 +850,7 @@ certgen:
   image:
     override: ~
     repository: "giantswarm/cilium-certgen"
-    tag: "v0.1.8"
+    tag: "v0.1.8@sha256:4a456552a5f192992a6edcec2febb1c54870d665173a33dc7d876129b199ddbd"
```

Comment:
digests might change depending on the repo:

```
$ skopeo inspect docker://quay.io/cilium/certgen:v0.1.9 | jq .Digest                                                                
"sha256:89a0847753686444daabde9474b48340993bd19c7bea66a46e45b2974b82041f"                                                               
$ skopeo inspect docker://quay.io/giantswarm/cilium-certgen:v0.1.9 | jq .Digest                                                     
"sha256:89a0847753686444daabde9474b48340993bd19c7bea66a46e45b2974b82041f"                                                               
$ skopeo inspect docker://docker.io/giantswarm/cilium-certgen:v0.1.9 | jq .Digest                                                   
"sha256:7d2d0adf80a7ef0eab29617ec4c30808d7f4ab0c0afd7916d62d3fc64caf6b73"                                                               
```

note docker.io is different from quay.io. 
I was not able to find exactly why but in the past we had problems with nginx pinned images by digest; so I'd say we drop it.

Reply:
FYI: it's not about the specific image version:
```
g:~ $ skopeo inspect docker://docker.io/giantswarm/cilium-certgen:v0.1.8 | jq .Digest
"sha256:1468b29d9e58def3e502291977c3a9faeca2d63de67a3910ebaed5c9fab5b22a"
g:~ $ skopeo inspect docker://quay.io/giantswarm/cilium-certgen:v0.1.8 | jq .Digest
"sha256:4a456552a5f192992a6edcec2febb1c54870d665173a33dc7d876129b199ddbd"
```

Reply:
But that would mean having to manually remove it after syncing :/ 
I'm going to test the app and see if it has issues. If not, we can keep it.